cypress/support/e2e.js

```diff
@@ -31,20 +32,24 @@ before(() => {
       localeSettingsId = body.id;
       localeSettingsValue = body.value;
       const localeValue = localeSettingsValue.match(/"locale":"(.*?)"/);
-      if (localeValue) {
-        const extractedLocale = localeValue[1];
-        Cypress.env('localeValue', extractedLocale);
-        if (extractedLocale !== defaultTestLanguage) {
-          const updatedValue = localeSettingsValue.replace(`"locale":"${extractedLocale}"`, `"locale":"${defaultTestLanguage}"`);
-          cy.putLocaleSettings(localeSettingsId, updatedValue);
-        }
+      const timezoneValue = localeSettingsValue.match(/"timezone":"(.*?)"/);
+      const extractedLocale = localeValue ? localeValue[1] : undefined;
+      const extractedTimezone = timezoneValue ? timezoneValue[1] : undefined;
+
+      Cypress.env('localeValue', extractedLocale);
+      Cypress.env('timezoneValue', extractedTimezone);
+
+      if (extractedLocale !== defaultTestLanguage || extractedTimezone !== defaultTestTimezone) {
```

Comment:
Does this definitely work if one of the two is null and the other is not? I'm always wary of combining this kind of logic inside an if condition

Reply:
Yes